src/main/java/io/github/debuggyteam/architecture_extensions/blocks/FacadeBlock.java

```diff
@@ -0,0 +1,31 @@
+package io.github.debuggyteam.architecture_extensions.blocks;
+
+import net.minecraft.block.AbstractLichenBlock;
+import net.minecraft.block.BlockState;
+import net.minecraft.block.LichenSpreadBehavior;
+import net.minecraft.block.ShapeContext;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.shape.VoxelShape;
+import net.minecraft.util.shape.VoxelShapes;
+import net.minecraft.world.BlockView;
+
+/**
+ * @author Gaming32 & Portal Cubed devs
+ * @since ${version}
```

Comment:
this annotation should always be frozen to one version, and not updated to the latest by gradle.

Reply:
Yeah, that kinda got carried over from here; it was never meant to be anything more than something that can be easily Ctrl+Shift+R then replaced when it was time to actually release the code.